Update layout.css

aaleksanyants opened the pull request:
If #jform_contact_email_copy gets float attribute, this causes below located div to get inline with it. I found this issue after enalbing captcha plugin which was completely floated moved to the right vertical border of the checkbox containing element.

aaleksanyants committed:
Update layout.css

If #jform_contact_email_copy gets float attribute, this causes below located div to get inline with it. I found this issue after enalbing captcha plugin which was completely floated moved to the right vertical border of the checkbox containing element.

brianteeman commented:
Testers please note that this is only for the beez template<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7385">issues.joomla.org/joomla-cms/7385</a>.</sub>

genesisfan commented:
I cannot reproduce this issue.
Please provide screenshots of this issue?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7385">issues.joomla.org/joomla-cms/7385</a>.</sub>

aaleksanyants commented:
![1](https://cloud.githubusercontent.com/assets/12964692/8636476/bfd0e868-2873-11e5-9937-bfd2f14ee447.PNG)
![2](https://cloud.githubusercontent.com/assets/12964692/8636477/c519abc0-2873-11e5-8737-190864cf7d83.PNG)

Please compare the bottom parts of these two screenshots

genesisfan commented:
Thank you @aaleksanyants did you use the Beez3 template or which else?

aaleksanyants commented:
@genesisfan , the pull request I made was regarding the change in Beez3's css file, so the screenshots I attach should also belong to the issue with Beez3 :-) Unfortunately I don't know if the same thing happens with other tamplates because I didn't check them.

genesisfan commented:
@aaleksanyants I was wrong sorry! I had selected protostar as template and not beez3.

Still...
without layout.css patch
![7385 without layout css patch](https://cloud.githubusercontent.com/assets/1644862/8644841/c9bfb79e-2942-11e5-99de-899a8c5051f2.jpg)

with layout.css patch
![7385 with layout css patch](https://cloud.githubusercontent.com/assets/1644862/8644848/d2a2c72a-2942-11e5-974f-a4cce1daf44f.jpg)

how it should be layout.css including another litte css customization:
![7385 how it should be](https://cloud.githubusercontent.com/assets/1644862/8644857/e8804c2a-2942-11e5-8f64-cfd0b7275123.jpg)

![7385 how it should be css](https://cloud.githubusercontent.com/assets/1644862/8644891/0edcd6ea-2943-11e5-871f-af1dc3e2ed3e.jpg)
Can make this modification?

aaleksanyants commented:
@genesisfan , thank you for your suggestion which produces better results, however removing the bottom margin causes 2 dives gluing ... The only right solution would be adding an empty div after the checkbox having clear='both' for its CSS style. This will break the floats rule and revert all following divs to the normal position. But this should be done in the output code somewhere in php, and I don't know where exactly...

aaleksanyants committed:
Update layout.css

The second change is done to always break label and field in the login form in two rows. Before this change if either login or password in translation did not fit in the row with the appropriate form field, this caused one of them to show inline while other to show in two rows, ie:
login [login_field]
password
[password_field]

aaleksanyants commented:
The second change is done for login form display under translated interface. In current version there was no rule to display the fields from new line and thus, if label and field of either login or password could not fit in one line, this caused them looking differently, ie:
Login [login_field]
Password
[password_field]

afberlin commented:
i did some testing and the position of captcha is wrong, the solution is indeed:
![solution](https://cloud.githubusercontent.com/assets/1644862/8644891/0edcd6ea-2943-11e5-871f-af1dc3e2ed3e.jpg "solution")<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7385">issues.joomla.org/joomla-cms/7385</a>.</sub>

fcschippers commented:
I've tried to reproduce this problem; with Joomla 3.5.0 Beta (12/12/2015) the problem can't be reproduced.

The Captcha label isn't in the right place.

A new pull request has been submitted:
 Update layout.css regarding Captcha label field #8658 
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7385">issues.joomla.org/joomla-cms/7385</a>.</sub>

roland-d commented:
@aaleksanyants thanks for your contribution. I am going to close this PR as we have a new PR #8658 that fixes the label field as well.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7385">issues.joomla.org/joomla-cms/7385</a>.</sub>
